codecov.yml

```diff
@@ -0,0 +1,2 @@
+codecov:
+  branch: main # set new Default branch
```

Comment:
FYI `main` is already the default branch of this repo, so this is not explicitly needed. But I am not against stating it explicitly here. Based on the docs this is just to select which branch the yaml will be taken from at codecov.io.

Reply:
In Codecov, the main branch was still incubation until today when we forced it to be main by setting a "global" YAML there. It wasn't possible to set it to main from the dropdown, we think that is because a report has never been uploaded for that branch (or any branch from this repo?) yet (for now, the main branch reports for opendatahub-io/opendatahub-operator go to Vaishnavi's Codecov...I think this is because the token used is from her account rather than an org one. We'll change that if needed, after talking to the dashboard folks to see if that's what they've got).